bin/deft

```diff
@@ -55,8 +75,8 @@ function getDestinationPath(fileName) {
   return path.join(process.cwd(), folder, fileName);
 }
 
-function pathExists(path) {
-  return (fs.existsSync || path.existsSync)(path);
+function pathExists(p) {
```

Comment:
Ugh! Can't believe this bug is still there! I actually think I fixed this locally on an old laptop and then never pushed... not that I have proof, mind you...